enterprise/app/workflows/workflows.tsx

```diff
@@ -459,7 +461,7 @@ class RepoItem extends React.Component<RepoItemProps, RepoItemState> {
       <div className="workflow-item container">
         <div className="workflow-item-column">
           <div className="workflow-item-row">
-            {this.state.runWorkflowActionStatuses && (
+            {this.props.isAdmin && this.state.runWorkflowActionStatuses && (
```

Comment:
nit: runWorkflowActionStatuses won't be populated since the "run" control is hidden, so we can leave this as-is too